metaflow/plugins/cards/card_modules/card.py

```diff
@@ -49,7 +74,8 @@ def _get_mustache(self):
         except ImportError:
             return None
 
-    def render(self, task) -> str:
+    # FIXME document data
+    def render(self, task, data=None) -> str:
```

Comment:
I think generally that we should:
- keep the same `render` API. We do need the `data` portion for runtime cards and could add a `get_data` which would return the latest data passed to `render_runtime`.
- the new methods `render_runtime`, `refresh` and `get_data` for example should be probably wrapped (or rather cards should implement a submethod). The wrapper should check whether the card is runtime render-able.

Basically, we should be able to say a card support runtime rendering in one place `RUNTIME_UPDATABLE` and the rest should come from there and we should catch errors that way (ie: the card says it is updatable but doens't provide a method or the other way around)